packages/node_modules/@webex/test-helper-test-users/src/index.js

```diff
@@ -79,50 +64,6 @@ function _create(options) {
   }
 }
 
-/**
- * Gets users from environment variable
- * @param {Object} options
- * @returns {Array<User>}
- */
-function _extractFromEnv(options) {
```

Comment:
This function will never be called because the test users library is now accessible by everyone. We can add this feature back in if it ever is requested.